$OpenBSD$

Add --enable-tor-browser-data-in-home-dir which implies
--enable-tor-browser-data-outside-app-dir and makes ~/.tor-browser
happen, c.f. patch-xpcom_io_TorFileUtils_cpp
--- configure.in.orig	Wed Sep  7 02:07:10 2016
+++ configure.in	Wed Sep 21 14:54:53 2016
@@ -6539,11 +6539,22 @@ if test -n "$TOR_BROWSER_UPDATE"; then
     AC_DEFINE(TOR_BROWSER_UPDATE)
 fi
 
+MOZ_ARG_ENABLE_BOOL(tor-browser-data-in-home-dir,
+[  --enable-tor-browser-data-in-home-dir
+                          Enable Tor Browser data in home dir],
+    TOR_BROWSER_DATA_IN_HOME_DIR=1,
+    TOR_BROWSER_DATA_IN_HOME_DIR= )
+
 MOZ_ARG_ENABLE_BOOL(tor-browser-data-outside-app-dir,
 [  --enable-tor-browser-data-outside-app-dir
                           Enable Tor Browser data outside of app directory],
     TOR_BROWSER_DATA_OUTSIDE_APP_DIR=1,
     TOR_BROWSER_DATA_OUTSIDE_APP_DIR= )
+
+if test -n "$TOR_BROWSER_DATA_IN_HOME_DIR"; then
+    TOR_BROWSER_DATA_OUTSIDE_APP_DIR=1
+    AC_DEFINE(TOR_BROWSER_DATA_IN_HOME_DIR)
+fi
 
 if test -n "$TOR_BROWSER_DATA_OUTSIDE_APP_DIR"; then
     AC_DEFINE(TOR_BROWSER_DATA_OUTSIDE_APP_DIR)
